lib/membrane_rtc_engine/endpoints/hls_endpoint.ex

```diff
@@ -148,7 +148,6 @@ if Enum.all?(
           tracks: %{},
           stream_beginning: nil,
           video_layout_state: nil,
-          stream_ids: MapSet.new(),
           video_layout_tracks_added: %{}
```

Comment:
As mentioned in our talk I think we should add a test for this whether we properly call functions from VideoLayoutMaker behavior.

Reply:
tests added